capstone/fitness/src/main/webapp/calendar.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<!-- HTML for calendar page. Containers load in information from java using the onload in body. -->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Your Calendar</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <!-- basic navbar for navigation during development. can be stylized later. -->
```

Comment:
nit: Please format comments like sentences (capitalization).

Also for comments like "can be stylized later", would recommend putting a new line and adding "TODO(piercedwilson): " in front so you remember to get back to it later.